TESTING.md

```diff
@@ -189,12 +218,10 @@ To run default Astyanax or CQL tests change the `test` property value in the abo
 
 #### TinkerPop Tests
 
-TinkerPop Thrift and CQL tests:
```

Comment:
(nitpick) I would leave that here and just remove the _CQL_ part. Otherwise this section only has a code block without any explanation whatsoever.